governance-exercises/VoteWizard_poll_exercise_01/README.md

```diff
@@ -0,0 +1,82 @@
+
+7. **Draft the Governance Polls:** The next step would be to draft the governance polls. The poll would include a brief overview of the proposal, clear voting options, and links to relevant resources for voters to make an informed decision. It's important to make sure that the text is impartial and without bias toward specific or general outcomes.
+
+8. **Check Grammar:** I would run the text through Grammarly to assure there are no spelling errors.
+
+9. **Review the Poll:** I would review the draft poll against the "poll creation checklist" to ensure it is in accordance with the guidelines and does not contain any errors or ambiguities. Then, I would create a PR to push changes to the main branch repo.
+
+10. **Communicate with Involved Parties:** I would share the poll outcome text with relevant parties before submitting it.
+
+11. **Poll Submission and Tagging:** I would follow the "poll submission checklist" to submit the finalized polls. While choosing the tags, I would evaluate the poll's impact using [this guide](https://manual.makerdao.com/governance/off-chain/impact-estimations), and then proceed to select other relevant tags.
+
+	- Given the proposed 50 million DAI increase to the Spork Protocol D3M Debt Ceiling and the onboarding of the ABC token (which inherently carries risk even if it had been onboarded to Maker core in the past), both polls would be tagged as high-impact.
+
+	- In this case, the tags would be the following:
+		- Poll 1: high-impact, risk-parameter, d3m, spork
+		- Poll 2: high-impact, collateral-onboard, risk-parameter, spork
+
+12. **Communicate the Polls:** After verifying the active status of all expected polls, I would communicate their availability on the #governance channel in the official MakerDAO discord. This action will promote widespread awareness among DAO members. Lastly, I would update the Governance Tracker sheet with the newly live polls.
+
+13. **Monitor and Address Queries:** Throughout the polling duration, I would actively monitor community engagement, answer questions, and address any concerns or confusion about the poll.
+
+## Assignment #2: Draft any forum responses required or implied by your course of action
+
+### The forum response I would make is:
+
+Hello, everyone!
+
+I've reviewed the proposal and am preparing to move forward with it. However, I wanted to share some updates regarding how we're going to handle this in the upcoming governance polls.
+
+Given the distinct nature of some parts of the proposal, I've made the decision to split the proposal into two separate polls. This allows the voters to consider and vote on each matter independently.
+
+These two polls will be presented during the next weekly governance cycle on Monday, July 24:
+
+1. Adjust Spork Protocol D3M Parameters and DAI Interest Rate Strategy:
+
+   - Increase of the Spork D3M Debt Ceiling from 50 million to 100 million DAI.
```

Comment:
You could also link the docs here. And/or remind the other authors in the thread that they should link to docs where they exist.